jd-geek-source/src/app/components/header/header.component.ts

```diff
@@ -0,0 +1,19 @@
+import { style } from '@angular/animations';
+import { Component, OnInit } from '@angular/core';
+import { colors } from '../../../constants/color';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+export class HeaderComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+    console.log(document.getElementById("headerbar"));
+    document.getElementById("headerbar").setAttribute("style", "background-color:" + colors.white);
```

Comment:
you should just add the background color via css instead of using javascript.

Reply:
I will be changing this.  The geek source instructions said to put the colors into a js constants file and I didn't see any other way to use them in CSS without doing it this way.  I have since figured out how to use them from a constant file in Sass.